### ISSUE TITLE
Not able to get undo + redo to work after editing text in a cell.

### ISSUE THREAD

**Issue by markwindsorr** [datalayer/jupyter-ui#292]
## Description

Pulling the latest version, I'm not able to get `CTRL Z` or `CTRL Y` to work on any key strokes.  Older versions it would work for normal key strokes but wouldn't work for a cut or paste operation. If you were to cut a piece of code and did something else, the cut code would be gone. 

## Reproduce

1. Edit a cell, try to undo or redo your changes to the cell

**Comment by markwindsorr** [comment]
** Update ** I have a fix for it.

**Comment by echarles** [comment]
Thx for the report Mark. Is a PR needed for the fix?

**Comment by markwindsorr** [comment]
@echarles Yup, Im having issues running the react package when I pull the latest version. I had to do 2 steps in `NotebookCommands.ts` to get it to work.

1. ) Add a custom command for undo and redo to the command registry. the `NotebookActions.undo(notebookPanel.content)` isn't not working as intended. so the else statement can be removed here. Same goes for redo.

![image](https://github.com/user-attachments/assets/a7f62b1e-2cda-4bc7-8832-243b2c0d2e62)

2. ) Removing .jp-mod-commandMode:focus from the selector in the bindings array

![image](https://github.com/user-attachments/assets/73b11fe8-635b-4ecb-aefa-b6c6c13d5fe7)

This may not be the best solution but it solves the issue of losing code once you cut something from a cell and lose it from the clipboard and want to get it back.

**Comment by echarles** [comment]
Undo/Redo has incurred changes since a few JupyterLab releases. If you have a fix like you explained, happy to review a PR :)
